docs/docs/reference/server-config.rst

```diff
@@ -118,6 +119,13 @@ log-file
 
     Default: (empty)
 
+log-format
+    Format string for console messages logged to file (see Python
```

Comment:
1. Please be more clear that `log-file` needs to be enabled for this to work
2. Please list all available fields and what they do here

Reply:
> 1. Please be more clear that `log-file` needs to be enabled for this to work

I've added something.

> 2. Please list all available fields and what they do here

There are way too many, I've opted for a proper link to upstream documentation.